### PR TITLE
Added some tests :)

### PR DESCRIPTION
**Pull request by FinalAngel**
Yay

### CODE REVIEW

File: tests/test_migrations.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            call_command(
+                'makemigrations',
+                interactive=False,
+                dry_run=True,
+                exit_code=True,
+                stdout=output,
+            )
+        except SystemExit as e:
+            # The exit code will be 1 when there are no missing migrations
+            assert unicode(e) == '1'
```
Comment (czpython) [inline tests/test_migrations.py]: likely not py3 compatible

File: tests/test_grid.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+)
+from djangocms_bootstrap4.contrib.bootstrap4_grid.constants import (
+    GRID_CONTAINERS,
+    GRID_ROW_VERTICAL_ALIGNMENT,
+    GRID_SIZE,
+)
+from djangocms_bootstrap4.constants import TAG_CHOICES
+
+
+class ContainerPluginTestCase(CMSTestCase):
```
Comment (czpython) [inline tests/test_grid.py:20]: please add `test_add` prefix to all these tests, otherwise it is unclear what this is testing

**Review by czpython — CHANGES_REQUESTED** [review]

File: tests/test_grid.py (hunk 2 above)
Comment (FinalAngel) [reply tests/test_grid.py:20]: @czpython like ``test_add_ContainerPluginTestCase``?

File: tests/test_migrations.py (hunk 1 above)
Comment (FinalAngel) [reply tests/test_migrations.py]: changing it to ``assert(unicode(e) == '1')``

File: tests/test_grid.py (hunk 2 above)
Comment (czpython) [reply tests/test_grid.py:20]: I meant adding the test_add to each method on the class

File: tests/test_migrations.py (hunk 1 above)
Comment (czpython) [reply tests/test_migrations.py]: I meant the unicode part.
checkout the six package (bundled in django).

**Review by czpython — APPROVED** [review]

**Pull request merged**